jaseci_core/jaseci/actions/live_actions.py

```diff
@@ -110,6 +110,13 @@ def load_module_actions(mod, loaded_module=None):
                     del live_actions[i]
 
         mod = importlib.import_module(mod)
+        try:
+            if hasattr(mod, "setup"):
+                mod.setup(**ctx)
+        except Exception:
+            logger.error(
+                f"Cannot run setup for module {mod}. Refer the setup parameters."
```

Comment:
I don't think this flow works for existing modules that don't have a setup().  We should first check if ctx is specified, if not, and only attempt to call setup() if there is a non empty ctx specified.

Reply:
Lets also make this error message more clear. Lets say this
```
Cannot run set up for module {mod}. This could be because the module doesn't have a setup procedure for initialization, or wrong set up parameters are provided.
```

Reply:
According to the current logic, it checks whether there is a setup function in the module. if so runs the setup module. with the context given. if the context is empty still it will run the setup function but with the default values given in the module allowing the module to do just one intialization, so we can remove the setup call in the module file.